BibBuddy/app/src/main/java/de/bibbuddy/NoteDao.java

```diff
@@ -252,6 +252,8 @@ public boolean linkNoteWithBook(Long bookId, Long noteId) {
    * @return returns the notes text value
    */
   public String findTextById(Long id) {
+    NoteTypeLut noteType = NoteTypeLut.TEXT;
```

Comment:
```suggestion
```

Reply:
line 186 is also not needed  `if (!notesList.isEmpty()) {` because if the size is 0 the loop will not be executed.

Reply:
I tried your suggestions and the app crashes every time. For me, it works well now. If you want to use this variable, then please provide me an example of how to use it. By calling the NoteTypeLut.TEXT the id of the notes is set to 1(TEXT). This is also explained in multiple sources about enum constructors. If you want then you can write your changes here or push them in the branch.

Reply:
You don't know what I mean. Let's have a meeting sometime and discuss it there.